app/js/index.js

```diff
@@ -27,6 +34,15 @@ var str2ab = function(str) {
   return buf;
 };
 
+var getIndexByValue = function( element, value ) {
```

Comment:
Since we're using jquery from bootstrap.
Can we use jquery function here as well?  http://stackoverflow.com/questions/13556941/get-index-of-selected-option-with-jquery.
we might also need to insert jquery to `bower.json`.